src/main/java/com/internet/shop/model/User.java

```diff
@@ -28,6 +29,22 @@ public User(Long id, String name, String login, String password) {
         this.password = password;
     }
 
+    public User(Long id, String name, String login, String password, byte[] salt) {
+        this.id = id;
+        this.name = name;
+        this.login = login;
+        this.password = password;
+        this.salt = salt;
+    }
+
```

Comment:
![image](https://user-images.githubusercontent.com/50640794/94562728-5aebe800-026e-11eb-9e95-6c2c4cccb321.png)

Reply:
make the same with your 4 constructors

Reply:
done